docs/src/cfe_sb.dox

```diff
@@ -508,31 +508,29 @@
            How many copies of the message are performed in a typical message delivery?
         </B><TR><TD WIDTH="5%"> &nbsp; <TD WIDTH="95%">
            There is a single copy of the message performed during a typical delivery.
-           During the #CFE_SB_SendMsg API, the software bus copies the message from the
-           callers memory space to the software bus memory space. The #CFE_SB_RcvMsg API
-           gives the user a pointer to the message in the software bus memory space. This
+           When transmitting a message, the software bus copies the message from the
+           callers memory space into a buffer in the software bus memory space.
+           The #CFE_SB_ReceiveBuffer API gives the user back a pointer to the buffer. This
            is equivalent to the copy mode send and pointer mode receive in the heritage
            software bus used on WMAP, ST5, SDO etc.
```

Comment:
Suggest updating the last sentence of this answer to compare the new approach to previous versions of cFE instead of "WMAP, ST5, SDO, etc." -> I think more users would be familiar with older versions of cFE than with those specific missions.

Reply:
Updated, see 76a0951.